cloudify/dispatch.py

```diff
@@ -288,15 +287,17 @@ def dispatch_to_subprocess(self):
                 os.environ[constants.BYPASS_MAINTENANCE] = 'True'
             env = self._build_subprocess_env()
 
-            plugin_dir = self._extract_plugin_dir()
-            if not plugin_dir and self._should_install_plugin():
-                self._install_plugin()
+            if not self._should_install_plugin():
```

Comment:
todo flip this condition, as soon as i figure out a better name than `should_install_plugin`